rpcs3/rpcs3qt/breakpoint_list.cpp

```diff
@@ -46,7 +49,10 @@ void breakpoint_list::ClearBreakpoints()
 
 void breakpoint_list::RemoveBreakpoint(u32 addr)
 {
-	m_breakpoint_handler->RemoveBreakpoint(addr);
+	if (!m_breakpoint_handler->RemoveBreakpoint(addr))
+	{
+		return;
```

Comment:
Don't return here. You won't be able to remove breakpoints without starting another game if you are stopped.

Reply:
Okay.